latex/ledger-spec.tex

```diff
@@ -478,7 +486,7 @@ \subsection{Deposits and Refunds}
       \fun{dvalue} & \PrtclConsts \to \DCert \to \Coin
         & \text{deposit amount of a certificate}\\
 
-      \fun{decay} & \PrtclConsts \to \mathbb{N}\times\mathbb{Q}^{+}
+      \fun{decay} & \PrtclConsts \to \lbrack 0, 1\rbrack\times\mathbb{Q}^{+}
```

Comment:
One thing that worries me is the fact that we're using real numbers to calculate the decay. Would this be a problem if these rules are implemented by different clients written in different programming languages, or using different real implementations? (float, double, etc).

Reply:
Yes, those are reasonable concerns.  But we are stuck with real numbers if we want exponential decay.  Should we ask the researchers if we can alternatively use some kind of linear decay?

Reply:
That sounds like a reasonable thing to do. But maybe we want to run this through Philipp and Duncan first. It might be just me that doesn't believe in real numbers :P

Reply:
sounds great!  It's not just you, I'm on team Kronecker too :)